charts/probate-frontend/values.template.yaml

```diff
@@ -0,0 +1,416 @@
+nodejs:
+  applicationPort: 3000
+  cpuRequests: 0.2
```

Comment:
Can you also remove this? The default of `25m` is fine.  We are trying to pack the nodes a little denser..

Reply:
Think I pinched that from cmc-citizen-frontend. Ive removed.